package/MDAnalysis/lib/distances.py

```diff
@@ -1096,9 +1096,9 @@ def _nsgrid_capped_self(reference, max_cutoff, min_cutoff=None, box=None,
             gridsearch = FastNS(max_cutoff, reference, box=box)
             results = gridsearch.self_search()
 
-        pairs = results.get_pairs()[::2, :]
+        pairs = results.get_pairs()
```

Comment:
This looks odd, but `self_search()` used to add each pair twice, then we'd return every other pair